Ricochet/Assets/_Scripts/PowerUp.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Enumerables;
+using UnityEngine;
+
+public class PowerUp : MonoBehaviour
+{
+    #region Inspector Variables
+    [Tooltip("Which type of powerup this is")]
+    [SerializeField]
+    private EPowerUp powerUpType;
+    [Tooltip("Drag the PowerUp Manager here")]
+    [SerializeField]
```

Comment:
This is something to talk about: Do we want other objects to have direct access to any other object besides the game manager? I understand the point of the powerup manager, but in this class it seems a little weird to access both managers. Scoping issues and such